Replicator/tests/ReplicatorCollectionSGTest.cc

```diff
@@ -722,6 +722,97 @@ TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Resolve Conflict SG", "[.SyncServe
     }
 }
 
+TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Auto Purge Enabled - Filter Revoked Revision - SGColl", "[.SyncServerCollection]") {
+    _authHeader = "Basic c2d1c2VyOnBhc3N3b3Jk"_sl; // Admin user for REST requests
```

Comment:
As we seem to need this in multiple tests, it's better to have it as a function that tests can be called.

Reply:
Or if this is required whenever we call SG REST API, it could be automatically set in the test's setup?